gossip/minimal/goshawk.go

```diff
@@ -367,6 +371,9 @@ func NewBoundaryGoshawk(_ context.Context, cfg *configpb.GoshawkConfig, hcLog, h
 	if fetchOpts.FlushInterval == 0 {
 		fetchOpts.FlushInterval = 10 * time.Minute
 	}
+	if fetchOpts.Reporter == nil {
+		fetchOpts.Reporter = &incident.LoggingReporter{}
```

Comment:
Please document that if `fetchOpts.Reporter` is nil, a LoggingReporter will be used.